### PR TITLE
fix: workaround for chrome issue

### PR DESCRIPTION
**Pull request by slavik-lvovsky**

### CODE REVIEW

**Review by ira-gordin-sap — APPROVED** [review]
looks ok for me, but I am not an expert in this staff

File: packages/frontend/src/App.vue [hunk 1]
```diff
@@ -664,6 +666,13 @@ export default {
   },
   mounted() {
     this.init();
+    const refreshId = setInterval(() => {
```
Comment (alex-gilin) [inline packages/frontend/src/App.vue:669]: please add a 'TODO' comment to remove that stuff when a root cause will solved

File: packages/frontend/src/ExploreGensApp.vue [hunk 2]
```diff
@@ -298,6 +299,15 @@ export default {
     ]);
     this.ready = true;
   },
+  mounted() {
+    const refreshId = setInterval(() => {
```
Comment (alex-gilin) [inline packages/frontend/src/ExploreGensApp.vue:303]: please add a 'TODO' comment to remove that stuff when a root cause will solved

**Review by alex-gilin — CHANGES_REQUESTED** [review]

File: packages/frontend/src/utils.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export default {
+  removeClass: (elementId, className, interval = 100) => {
```
Comment (alex-gilin) [inline packages/frontend/src/utils.js:2]:
```suggestion
  removeCssClassWhenRendered: (elementId, className, interval = 100) => {
```

File: packages/frontend/src/App.vue [hunk 4]
```diff
@@ -664,6 +667,7 @@ export default {
   },
   mounted() {
     this.init();
+    utils.removeClass("workaround", "material-icons");
```
Comment (alex-gilin) [inline packages/frontend/src/App.vue:670]: please consider to add a 'TODO' comment to remove that stuff when root cause will solved

File: packages/frontend/src/ExploreGensApp.vue [hunk 5]
```diff
@@ -298,6 +303,9 @@ export default {
     ]);
     this.ready = true;
   },
+  mounted() {
+    utils.removeClass("workaround-ex", "material-icons");
```
Comment (alex-gilin) [inline packages/frontend/src/ExploreGensApp.vue:307]: please consider to add a 'TODO' comment to remove that stuff when root cause will solved

**Review by alex-gilin — CHANGES_REQUESTED** [review]

**Review by alex-gilin — APPROVED** [review]

**Pull request merged**